docs/src/site/twiki/FalconDocumentation.twiki

```diff
@@ -615,10 +617,35 @@ Failure to follow any of the above rules would result in a process submission fa
 present in the system for the specified time period, the process can be submitted and scheduled, but all instances
 created would result in a WAITING state unless data is actually provided in the cluster.
 
+---++ Updating cluster entity definition
+User would want to update the cluster entity in following scenarios
```

Comment:
How about combining the first two sentences: Cluster entities can be updated when the user wants to change their interface endpoints or properties, e.g. hadoop clusters updated from unsecure to secure; hadoop cluster moved from non high-availability to high-availability, etc.